docs/hw/jetson_hookup.md

```diff
@@ -1,32 +1,86 @@
 # iRobot® Create® 3 Hookup Guide for NVIDIA® Jetson™
 
+The iRobot® Create® 3 robot has a JST XH-style connector on its internal [Adapter Board](./adapter.md) which is capable of supplying a **maximum of 2A** of curret at the current battery volrage (14.4V nominal).
```

Comment:
"voltage"

Reply:
"current"

Reply:
Please put spaces between numbers and units.

Reply:
[9649f15](https://github.com/iRobotEducation/create3_docs/pull/385/commits/9649f15d06c9403b943bb248ce189fd2a19ca424)